pkg/proc/bininfo.go

```diff
@@ -332,7 +333,7 @@ func FindFunctionLocation(p Process, funcName string, lineOffset int) ([]uint64,
 
 	if lineOffset > 0 {
 		fn := origfns[0]
-		filename, lineno := fn.cu.lineInfo.PCToLine(fn.Entry, fn.Entry)
+		filename, lineno, _ := bi.PCToLine(fn.Entry)
```

Comment:
What about adding a `func (*Function) pcToLine` method that does the right thing?

Reply:
Just not re-doing the lookup of fn. More of an aesthetic preference than anything. If you don't like it it's also fine.

Reply:
Ack, refactored a bit to remove the double lookup.